cmd/podman/cp.go

```diff
@@ -223,8 +228,23 @@ func copy(src, destPath, dest string, idMappingOpts storage.IDMappingOptions, ch
 	copyWithTar := chrootarchive.CopyWithTarAndChown(chownOpts, digest.Canonical.Digester().Hash(), idMappingOpts.UIDMap, idMappingOpts.GIDMap)
 	untarPath := chrootarchive.UntarPathAndChown(chownOpts, digest.Canonical.Digester().Hash(), idMappingOpts.UIDMap, idMappingOpts.GIDMap)
 
-	if srcfi.IsDir() {
+	filename := filepath.Base(destPath)
+	if filename == "-" && !isFromHostToCtr {
+		file, err := os.Open(srcPath)
+		if err != nil {
+			return errors.Wrapf(err, "error opening file %s", srcPath)
+		}
+		defer file.Close()
+
+		data, err := ioutil.ReadAll(file)
+		if err != nil {
+			return errors.Wrapf(err, "error streaming tar file to stdout")
+		}
+		fmt.Print(string(data))
```

Comment:
This doesn't seem safe - we need to do a `Write()` of the raw bytes to `os.Stdout` - I don't know what formatting `Print` does, but it's almost certainly potentially hazardous to the bytes in transit